server/src/core/refresh-token/refresh-token.entity.ts

```diff
@@ -23,7 +23,9 @@ export class RefreshToken {
   @PrimaryGeneratedColumn('uuid')
   id: string;
 
-  @ManyToOne(() => User, (user) => user.refreshTokens)
+  @ManyToOne(() => User, (user) => user.refreshTokens, {
```

Comment:
Delete a user should delete all its refresh tokens. Otherwise the delete will fail if refresh tokens exist due to the foreign key constraint.